src/Acf.php

```diff
@@ -185,4 +199,23 @@ private static function apply_default_transform( $field_obj ) {
 	private static function get_transform_class_name( $field_type ) {
 		return '\\' . __NAMESPACE__ . '\\Acf\\Transforms\\' . str_replace( '_', '', ucwords( $field_type, '_' ) );
 	}
+
+	/**
+	 * Get an ACF group
+	 *
+	 * @param $group_id
+	 * @return bool
+	 */
+	private static function get_group( $group_id ) {
+		// @codingStandardsIgnoreStart
+		// Ignore use WP_Query rule because we don't know if this will be a sub-query and hence create complications.
```

Comment:
Nice solution!
However, I'm not sure if it's more efficient, because the function has to parse multi-level url's, xx/xxx/xxx. Also it fails the code sniff and I'm not sure it's possible to just exlude this function https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/4e3a53c6e211385fb16512cb32d4f091cbda4fe8/WordPress/Sniffs/VIP/RestrictedFunctionsSniff.php

````
get_page_by_path is prohibited, please use
     |       |     wpcom_vip_get_page_by_path() instead.